templates/paperlessngx/index.ts

```diff
@@ -0,0 +1,64 @@
+import { Output, randomPassword, Services } from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const secretkey = randomPassword();
```

Comment:
there should be a helper called `randomString()` - you can use that instead of `randomPassword()`

Reply:
randomstring works just as well, - issues whatsoever

Reply:
I meant randomPassword();

Reply:
fixed